docs/changes.rst

```diff
@@ -1247,7 +1247,7 @@ data in a local SQLite database and will reuse it for all tests of the same
 type.
 
 LICENSING CHANGE: Hypothesis is now released under the Mozilla Public License
-2.0. This applies to all versions from 0.4.0 onwards until further notice.
+2.0. This applies to all versions from 0.4.0 on wards until further notice.
```

Comment:
There are a few others in the diff where it looks like the spellchecker has "corrected" something that was already fine -- the diff was quite long, so I'm not going to check them all myself.